tests/test_ops/test_voxelization.py

```diff
@@ -140,6 +140,10 @@ def test_voxelization_nondeterministic():
 
 
 @pytest.mark.parametrize('device_type', [
+    pytest.param(
+        'cuda:0',
+        marks=pytest.mark.skipif(
+            not IS_CUDA_AVAILABLE, reason='requires CUDA support')),
```

Comment:
Why do we need to add this test which has been tested in https://github.com/open-mmlab/mmcv/pull/2790/files#diff-92c8d3007d5a6331f491492485d0ab27e088ed6088bef9b0e228ea0adfc7248cR22

Reply:
this is for mlu self test, dipu will mock its to cuda device, so here add `device = 'cuda:0',`(so we can reuse testcase called test_voxelization_mlu). we will not used above testcase `test_voxelization` because this is only used for real cuda device.

Reply:
Need to add a comment for this